docs/state.md

```diff
@@ -236,7 +236,47 @@ AppStore
     'tabs.switch-to-new-tabs': boolean, // true if newly opened tabs should be focused immediately
     'tabs.tabs-per-page': number // number of tabs per tab page
   },
-  sites: {
+  // TODO sort alphabetically
```

Comment:
Regarding sort, I was thinking of creating simple sort object that would have siteKey and order key. This way we can have bookmarks and folders split, but still keep the order that we need for the bookmark toolbar. Something similar we use for tabs as well.

What do you think?